kura/distrib/src/main/resources/Win32-nn/Kura.nsi

```diff
@@ -0,0 +1,435 @@
+!include "Library.nsh"
```

Comment:
Beside not having a copyright header, this file is copied from somewhere else [1]. For external content you will need to file a CQ.

[1] http://nsis.sourceforge.net/mediawiki/index.php?title=ReplaceInFile&oldid=20490